rc/tools/spell.kak

```diff
@@ -87,57 +87,60 @@ define-command spell-clear %{
     unset-option buffer spell_regions
 }
 
-define-command spell-next %{ evaluate-commands %sh{
+define-command spell-rel-jump -hidden -params 0..1 %{ evaluate-commands %sh{
     anchor_line="${kak_selection_desc%%.*}"
     anchor_col="${kak_selection_desc%%,*}"
     anchor_col="${anchor_col##*.}"
 
-    start_first="${kak_opt_spell_regions%%|*}"
-    start_first="${start_first#* }"
+    spell_first="${kak_opt_spell_regions%%|*}"
+    spell_first="${spell_first#* }"
+    spell_last="${kak_opt_spell_regions##* }"
+    spell_last="${spell_last%|*}"
 
     # Make sure properly formatted selection descriptions are in `%opt{spell_regions}`
-    if ! printf %s "${start_first}" | grep -qE '^[0-9]+\.[0-9]+,[0-9]+\.[0-9]+$'; then
+    if ! printf %s "${spell_first}" | grep -qE '^[0-9]+\.[0-9]+,[0-9]+\.[0-9]+$'; then
         exit
     fi
 
-    printf %s "${kak_opt_spell_regions#* }" | awk -v start_first="${start_first}" \
-                                                  -v anchor_line="${anchor_line}" \
-                                                  -v anchor_col="${anchor_col}" '
-        BEGIN {
-            anchor_line = int(anchor_line)
-            anchor_col = int(anchor_col)
-        }
-
-        {
-            for (i = 1; i <= NF; i++) {
-                sel = $i
-                sub(/\|.+$/, "", sel)
+    if [ "$1" = "-rev" ]; then
```

Comment:
The FS approach seems good.

Here we should initialize `rev`, for the (unlikely) case where we inherit it from the environment